tests/python/pants_test/backend/native/subsystems/test_native_toolchain.py

```diff
@@ -103,7 +104,7 @@ def _hello_world_source_environment(self, file_name, contents, scheduler_request
       with safe_open(source_file_path, mode='wb') as fp:
         fp.write(contents)
 
-      products, subjects = zip(*scheduler_request_specs)
+      products, subjects = list(zip(*scheduler_request_specs))
```

Comment:
@cosmicexplorer I'm not sure what this line is being used for - IntelliJ warns `products, subjects` is never used. Is it safe to remove this line?